heimdall.google-addon/execution.html

```diff
@@ -16,72 +16,95 @@
             crossorigin="anonymous"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
 </head>
-
 <script>
-    
-    var list_of_ids;
-    
-    function env(value){
-       if(value == 'Test-onDemand'){
-           document.getElementById('addbranch').style="visibility: display; height: 50px;width: 300px;";      
-       }
-       else{
-            document.getElementById('addbranch').style="visibility: hidden; height: 0px;width: 0px;";
-       }
-    }
-    
-    function fun(){
-        var env=$("#environment").val();
-        var filter=$("#selectFilter").val();
-        var suites;
-        var tags;
-        if(filter == "suite"){
-           suites=$("#suitesMultiSelect").val();
-           tags = list_of_ids;
-        }else{
-           tags = "@" + filter;
-        }
+
+   var list_of_ids;
+
+   function env(value) {
+      if (value == 'Test-onDemand') {
+         document.getElementById('addbranch').style = "visibility: display; height: 50px;width: 300px;";
+      }
+      else {
+         document.getElementById('addbranch').style = "visibility: hidden; height: 0px;width: 0px;";
+      }
+   };
+
+
+   function fun() {
+      var customEnv = document.getElementById("customEnv");
+      var env = (customEnv && customEnv.value === "") ? $("#environment").val() : customEnv.value;
```

Comment:
why are we using two type of id fetch? One via `document.getElementById` and another by `$("#environment")` . Can't we stick to one way of getting value by id?

Reply:
@tan31989 Changed everything to document.getElementById